src/view/form/editing-form-template.js

```diff
@@ -0,0 +1,113 @@
+import{formatSrtingToDateTime, capitalize } from '../../utils.js';
+import { TYPES } from '../../const.js';
+
+function showType(types, type) {
+  return types.map((item) => (` <div class="event__type-item">
```

Comment:
здесь я бы тоже как то по понятнее параметры назвал, например 
```suggestion
function showType(types, activeType) {
  return types.map((item) => (` <div class="event__type-item">
```